docs/data/pages.ts

```diff
@@ -264,68 +264,160 @@ const pages: MuiPage[] = [
         pathname: '/x/api/date-pickers',
         title: 'API Reference',
         children: [
-          { pathname: '/x/api/date-pickers', title: 'Index' },
-          { pathname: '/x/api/date-pickers/date-calendar', title: 'DateCalendar' },
-          { pathname: '/x/api/date-pickers/date-picker', title: 'DatePicker' },
+          { pathname: '/x/api/date-pickers/date-calendar/', title: 'DateCalendar' },
```

Comment:
@LukasTy I added the missing pages and sorted them alphabetically

We only miss `PickerStaticWrapper` which is going away on v6 :+1:

Reply:
There is some sort of an issue with routing/path setup as going to any of the specific API doc pages does not mark that page as active.

Reply:
Fixed :+1: